data-entry/invoice-editor/invoice-editor-flow/src/main/java/com/vaadin/invoice/editor/InvoiceEditor.java

```diff
@@ -174,6 +185,10 @@ public InvoiceEditor() {
         add(controlsLine, board, addsLine, grid, detailsLine);
     }
 
+    public static void scrollTo(GridPro<?> grid, int index) {
+        UI.getCurrent().getPage().executeJavaScript("$0._scrollToIndex(" + index + ")", grid.getElement());
```

Comment:
We are avoiding usage of webcomponent's private API.
Moreover, it's not properly working in this case.